index.html

```diff
@@ -277,7 +277,7 @@ <h3>Fossasia Attracting Students</h3>
                 <h3> FOSSASIA | Projects </h3>
             </div>
         </div>
-        <div class="row v-align-children">
+        <div class="row v-align-children"style="padding-bottom: 25px">
```

Comment:
Do not hardcode style element in html, rather give it a new class and make relevant changes in CSS file for the defined class.